share/shares.go

```diff
@@ -1,5 +1,5 @@
-// Package shares contains the Share data structure.
-package shares
+// package share contains the Share data structure.
```

Comment:
[optional][follow-up] we could expand this to:

```suggestion
// Package share contains the data structures: share, namespace, blob.
```